maskrcnn_benchmark/modeling/detector/generalized_rcnn.py

```diff
@@ -27,8 +28,12 @@ def __init__(self, cfg):
         super(GeneralizedRCNN, self).__init__()
 
         self.backbone = build_backbone(cfg)
-        self.rpn = build_rpn(cfg)
-        self.roi_heads = build_roi_heads(cfg)
+        if cfg.MODEL.RETINANET_ON:
+            self.rpn = build_retinanet(cfg)
```

Comment:
I'm thinking about moving this logic into `build_rpn`, and the same for `build_roi_heads`, so that this part keeps the same.
What do you think?